ethcore/sync/src/chain/fork_filter.rs

```diff
@@ -0,0 +1,191 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+// Re-export eip-2124 crate contents here.
+pub use eip_2124::{BlockNumber, ForkId, RejectReason};
```

Comment:
I think the fact that we're re-exporting these types is an indication that we might want to move the `eip-2124` crate into a sub-crate to this crate?

I also wonder if it makes sense to rename it to just `forkid`? I know there's prior art to name crates `eip-xxxx` but I'm starting to think it's a bad practice.